devU-api/scripts/populate-db.ts

```diff
@@ -0,0 +1,170 @@
+
+//Returns the ID of the newly created entry
+async function SendPOST(path: string, requestBody: string) {
+    let response = await fetch(API_URL + path, {
+        method: "POST",
+        headers: {
+            "Authorization": "Bearer " + TOKEN,
+            "Content-Type": "application/json"
+        },
+        body: requestBody
+    })
+    if (response.status >= 400) {
+        throw new Error("400/500 Level HTTP Response Recieved")
+    } else {
+        const responseBody = await response.json()
+        console.log(responseBody)
+        return responseBody.id
+    }
+}
+
+/*
+    -- Billy --
+    Student in 302
+
+    -- Bob --
+    Student in 312
+
+    -- Jones --
+    Student in 302 & 312, hasn't submitted anything
+*/
+async function RunRequests() {
+    try {
+        //Users
+        const userBilly = await SendPOST("/users", JSON.stringify({
+            email: "billy@buffalo.edu", externalId: "billy", preferredName: "Billiam"
+        }))
+        const userBob = await SendPOST("/users", JSON.stringify({
+            email: "bob@buffalo.edu", externalId: "bob", preferredName: "Bobby"
+        }))
+        const userJones = await SendPOST("/users", JSON.stringify({
+            email: "jones@buffalo.edu", externalId: "jones", preferredName: "Jones"
+        }))
+
+
+        //Courses
+        const course312 = await SendPOST("/courses", JSON.stringify({
+            name: "Web Applications", semester: "s2024", number: "CSE312", startDate: "2024-01-24", endDate: "2024-05-10"
+        }))
+        const course302 = await SendPOST("/courses", JSON.stringify({
+            name: "Intro to Experiential Learning", semester: "s2024", number: "CSE302", startDate: "2024-01-24", endDate: "2024-05-07"
+        }))
+
+
+        //UserCourse
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userBilly, courseId: course302, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userBob, courseId: course312, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userJones, courseId: course302, level: "student", dropped: false
+        }))
+        SendPOST("/user-courses", JSON.stringify({
+            userId: userJones, courseId: course312, level: "student", dropped: false
+        }))
+        
+
+        //Assignments
+        const assign312_1 = await SendPOST("/assignments", JSON.stringify({
+            courseId: course312, name: "Homework 1", startDate: "2024-02-05", dueDate: "2024-02-26", endDate: "2024-03-11", gradingType: "code", categoryName: "Homework",
```

Comment:
Add times to the start/due/end timestamps for assignments since the standard usage will have times for all of these

Reply:
.. you do have times on some of the other assignments. Please add times to all of them. People might use this test data as a template/reference when building features so we want this data to look as real as possible